tests/README.md

```diff
@@ -46,3 +46,9 @@ Prefixing ENVs:
 ```bash
 SINDRI_API_KEY=your_api_key SINDRI_API_URL=https://sindri.app/api pytest
 ```
+
+For Sindri internal developers:
+- If you are running the sindri api locally at ~/forge with ~/forge/API_KEY file populated with a valid api key
```

Comment:
This sentence is unclear and seems unfinished.

Reply:
revised in [3c15efa](https://github.com/Sindri-Labs/sindri-python/pull/39/commits/3c15efaf9e96fdcb54210c4d98a424c4ad6d6d02)

Reply:
better, thanks